### ISSUE TITLE
Multi-Select Matrix With Different Cell types - A screen reader doesn't Announce Column Captions when tabbing through row cells

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-library#7274]
User Issue: T15336 - Matrix question types and screen reader
https://surveyjs.answerdesk.io/internal/ticket/details/T15336
***

You can run the NVDA screen reader and test the following demo: [Multi-Select Matrix](https://surveyjs.io/form-library/examples/questiontype-matrixdropdown/reactjs).
Unfortunately, when tabbing through row cells, corresponding column names/captions are not announced. The column caption is only announced for the Comment column type.
![image](https://github.com/surveyjs/survey-library/assets/22372972/66ac979a-7fcf-470e-b9de-00460a9bd4e2)
